devices/common_patches/h7_common_dualcore.yaml

```diff
@@ -454,12 +584,54 @@ RCC:
       CAMITFLPEN:
         name: DCMILPEN
         description: "DCMI peripheral clock enable during csleep mode"
+  AHB3ENR,C1_AHB3ENR:
+    _add:
+      DTCM1EN:
+        description: D1 DTCM1 block enable
+        bitOffset: 28
+        bitWidth: 1
+      DTCM2EN:
+        description: D1 DTCM2 block enable
+        bitOffset: 29
+        bitWidth: 1
+      ITCM1EN:
+        description: D1 ITCM block enable
+        bitOffset: 30
+        bitWidth: 1
+      AXISRAMEN:
+        description: AXISRAM block enable
+        bitOffset: 31
+        bitWidth: 1
```

Comment:
The RM says `This location is 'Reserved' for register RCC_C1_AHB3ENR` for bits 28-31, so perhaps these additions should be `AHB3ENR` only?

Reply:
That's how I interpret that footnote too. Removing from `C1_AHB3ENR`